### ISSUE TITLE
Document verbose and quiet options

### ISSUE THREAD

**Issue by TinCanTech** [OpenVPN/easy-rsa#703]
Option `-v|--verbose` only works with `show-expire` and `show-renew`.
It is only meant as a way to verify that:

        [ "$db_serial" = "$cert_serial" ] || return 0

is valid. Even though the code reads as `[ FALSE ] then return TRUE`.

Actual code:
```
		# db serial must match certificate serial, otherwise this
		# is an issued cert that replaces a renewed cert
		if [ "$db_serial" != "$cert_serial" ]; then
			verbose "* serial mismatch:
  db_serial:    $db_serial
  cert_serial:  $cert_serial
  cert_file_in: $cert_file_in"
			return 0
		fi
```

When `db_serial` does **not** match `cert_serial` that is expected. The mismatch is caused by having duplicate `commonName`s map to differing unique `serialNumber`s.  This is due to renewing certificates via any method.

**Comment by TinCanTech** [comment]
Also `-q|--quiet` to disable `information()`

Initial merge: 8b7e79096b18afc5c61bfbaee204c1f7401f0019

**Comment by TinCanTech** [comment]
Closed via fb198b17e29de739482a52b05f24f523d4d1c95c
